packages/crayons-core/src/components/menu/readme.md

```diff
@@ -0,0 +1,80 @@
+# Menu (fw-menu)
+
+## Usage
+
+Menu is a container for `fw-menu-item` components.
+
+```html live
+<template>
+  <fw-menu style="--menu-max-width: 300px">
+    <fw-menu-item
+      >Messages
+      <fw-icon name="email" size="16" slot="prefix"></fw-icon>
+      <div class="badge" slot="suffix">10</div>
+    </fw-menu-item>
+
+    <hr class="separator" />
```

Comment:
Shall we create a divider component like shoelace https://shoelace.style/components/divider
?

Reply:
Sure Parsu, will raise a PR for fw-divider component